### PR TITLE
chore: update dependencies for major Qwic update

### PR DESCRIPTION
**Pull request by totto2727**
Qwik v1 has been released. As a result, the library did not work with the `^0.100.0` specification in dependencies and peerDependencies. Therefore, I have updated to `^1.0.0`.

I have confirmed that it also works with `> 1.0.0` in the following repositories.
https://github.com/totto2727-org/qwik-with-hono

I use a translation tool and apologize if the translation is strange.

### CODE REVIEW

File: .changeset/four-ties-love.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@hono/qwik-city': patch
+---
+
+#94 81fe019: update qwic
```
Comment (yusukebe) [inline .changeset/four-ties-love.md:5]: qwic is Qwi**k**!

**Comment by yusukebe** [comment]
Hi @totto2727 !

Thank you for creating this PR, it looks great! But, there's a small typo I'd like you to correct before merging. "Qwik" has been misspelled as "Qwic". Please correct this and we'll be good to go!

**Comment by yusukebe** [comment]
@totto2727 

Hi! You should write Qwik instead of Qwic!

**Comment by totto2727** [comment]
@yusukebe 
Sorry to bother you over and over...

Fixed!

File: .changeset/four-ties-love.md [hunk 2]
```diff
@@ -2,4 +2,4 @@
 '@hono/qwik-city': patch
 ---
 
-#94 81fe019: update qwic
+#94 81fe019: update Qwic
```
Comment (yusukebe) [inline .changeset/four-ties-love.md:5]: @totto2727 

Hey again and again. It should be "Qwi**k**"!

Screenshot 2023-06-22 at 22 35 19

**Review by yusukebe — COMMENTED** [review]
Hi @totto2727 !

Can you fix it?

File: .changeset/four-ties-love.md (hunk 2 above)
Comment (totto2727) [reply .changeset/four-ties-love.md]: Sorry…
I forgot to git add…

**Comment by yusukebe** [comment]
@totto2727 

Thank you so much! I'll merge it.

**Pull request merged**